server/views/pages/notifications.ejs

```diff
@@ -60,7 +60,6 @@
         <% } %>
 
       </div>
-      <% if (!hmppsAuthMFAUser){%>
```

Comment:
Why is this hidden for `hmppsAuthMFAUser`?

Reply:
This may just be a preference of mine, but I like to keep any logic out of template files, as they are already so full of fluff, they're hard to read.

So I would pass something like `isNotValidUser` to this file and display on that logic.

 This may be a personal preference thing though. I think @jonhindle is the go to person on _how to template good._

Reply:
Its kind of a moot point as this is all going. It was also always all going too 🤔 And we reused the const that was already established.
My preference is for the more descriptive though in this instance, because the context is relevant on all pages that this might show up on. I think the generic name would have obfuscated both the nature of the exclusion and those who were being excluded.
What I don't like is the negative logic forcing me to think against my will 😀